### ISSUE TITLE
Number of global steps

### ISSUE THREAD

**Issue by Sashweed** [1228337123/tensorflow-seq2seq-chatbot#1]
Firstly thank you for the updated execute.py, I couldn't for the life of me get Suriya's script working but given Im very very new ubuntu, python and TF I though it was my fault, your git worked first time.

having said that this is probably such a stupid question but .... when does this stop (or how do I stop it cleanly)

I thought it might stop 

"if counter % 100000 == 0:" but

global step 101000 learning rate 0.4432 step-time 0.63 perplexity 2.78
  eval: bucket 0 perplexity 130.77
  eval: bucket 1 perplexity 161.28
  eval: bucket 2 perplexity 91.37
  eval: bucket 3 perplexity 185.74

...... maybe don't worry about the cleanly ... turns out trying to copy from a putty window sends a keyboard interrupt.

:(

Thanks again for your work on this.

J

**Comment by dinnid0** [comment]
have you found a solution? i need help about this topic as well : P
would be glad to hear your conclusions

**Comment by Sashweed** [comment]
Nope... Gave up on this Git and found a few others that were a little more mature.
